docs/api/Components.md

```diff
@@ -182,31 +157,29 @@ describe('TestComponent', function () {
             // React must be required after window is set
             React = require('react');
```

Comment:
@jedireza lab is a test framework like mocha, [jsx-test](https://github.com/3den/jsx-test) is just a s simple tool to help testing react components. 

It removes a lot of boilerplate code You can see it being used here => https://github.com/yahoo/fluxible-immutable-utils/blob/master/tests/unit/ComponentMixin.js